packages/tools/devtools/devtools-core/src/test/DefaultVisualizers.spec.ts

```diff
@@ -433,471 +495,252 @@ describe("DefaultVisualizers unit tests", () => {
-								"DefaultVisualizer_SharedTree_Test.child-item": {
-									children: {
-										object: {
-											children: {
-												childData: {
-													children: {
-														kind: {
-															value: "Optional",
-															typeMetadata: "string",
-															nodeKind: "ValueNode",
-														},
-														types: {
-															children: {
-																"0": {
-																	value: "com.fluidframework.leaf.string",
-																	typeMetadata: "string",
-																	nodeKind: "ValueNode",
-																},
-															},
-															nodeKind: "TreeNode",
-															typeMetadata: "object",
-														},
-													},
-													nodeKind: "TreeNode",
-													typeMetadata: "object",
-												},
-												childField: {
-													children: {
-														kind: {
-															value: "Value",
-															typeMetadata: "string",
-															nodeKind: "ValueNode",
-														},
-														types: {
-															children: {
-																"0": {
-																	value: "com.fluidframework.leaf.boolean",
-																	typeMetadata: "string",
-																	nodeKind: "ValueNode",
-																},
-																"1": {
-																	value: "com.fluidframework.leaf.handle",
-																	typeMetadata: "string",
-																	nodeKind: "ValueNode",
-																},
-																"2": {
-																	value: "com.fluidframework.leaf.string",
-																	typeMetadata: "string",
-																	nodeKind: "ValueNode",
-																},
-															},
-															nodeKind: "TreeNode",
-															typeMetadata: "object",
-														},
-													},
-													nodeKind: "TreeNode",
-													typeMetadata: "object",
-												},
-											},
-											nodeKind: "TreeNode",
-											typeMetadata: "object",
-										},
-									},
-									nodeKind: "TreeNode",
-									typeMetadata: "object",
+							},
+							nodeKind: "TreeNode",
+							tooltipContents:
+								'{ americano : com.fluidframework.leaf.boolean, bubbleTea : com.fluidframework.leaf.string, chaiLatte : DefaultVisualizer_SharedTree_Test.chai-latte-object, dalgona : DefaultVisualizer_SharedTree_Test.Array<["DefaultVisualizer_SharedTree_Test.dalgona-object"]>, espresso : DefaultVisualizer_SharedTree_Test.Array<["com.fluidframework.leaf.number","com.fluidframework.leaf.string"]> }',
+						},
+						baz: {
+							value: "128",
+							nodeKind: "ValueNode",
+							tooltipContents:
+								"com.fluidframework.leaf.number | com.fluidframework.leaf.string | com.fluidframework.leaf.boolean",
+						},
+						foobar: {
+							children: {
+								anthropology: {
+									value: "1",
+									nodeKind: "ValueNode",
+									tooltipContents:
+										"com.fluidframework.leaf.string | com.fluidframework.leaf.number | com.fluidframework.leaf.handle | DefaultVisualizer_SharedTree_Test.map-object",
```

Comment:
If you change the name you give to the schema factory to something shorter instead of "DefaultVisualizer_SharedTree_Test" (like just "Test"), you'll have much shorter strings to compare against in these tests.